tm2/pkg/std/memfile.go

```diff
@@ -50,12 +50,19 @@ var (
 // file names must contain dots.
 // NOTE: this is to prevent conflicts with nested paths.
 func (mempkg *MemPackage) Validate() error {
+	if mempkg == nil {
+		return errors.New(fmt.Sprintf("package is nil"))
```

Comment:
You should format errors with `fmt.Errorf`.

Looking at this again, you don't even format an error here, this entire call is useless - define it as a package-level error